docs/pr-review-guidelines.md

```
@@ -0,0 +1,80 @@
+
+## Code Quality Review
+
+- [ ] **Have you looked at all the code files?**
+
+### File-Level Checks 
+
+- [ ] **Every file looks sane** – this helps with anything our linter may not track, aim to be on the lookout for ways we can improve our CI to help avoid these cases in the future
+
+- [ ] **Verified eslint ignore statements** – sometimes we want these (intentional console log statements), most times we do not
+
+    - [ ] **No ignoring of React hook dependency errors** (there is rarely a good reason for this)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)